CHANGELOG

```diff
@@ -0,0 +1,109 @@
+# Changelog
+
+## [Unreleased]
+### New Features
+ * Support reapply configuration without link down when possible.
+ * Relicensed to LGPL 2.1+.
+ * Allow editing existing interface without type property in desire state.
+
+### Bug Fixes
+
+ * Allow adding slave to linux bridge with name only.
+ * Allow enabling IPv6 with empty address.
+ * netapplier: Initialize mainloop global object on failure.
+ * DNS: Preserve DNS config with empty state.
+
+## [0.0.7] - 2019-06-13
+### New Features
+ * Support editing DNS client configuration.
+ * Support editing IP routes.
+ * API: Promote the public API to libnmstate level:
+    * libnmstate.show()
+    * libnmstate.apply()
+    * libnmstate.commit()
+    * libnmstate.rollback()
+    * libnmstate.error
+    * libnmstate.schema
+
+### Bug Fixes
+
+ * netapplier: Optimize new and existing interface processing.
```

Comment:
I would drop the `netapplier`. How about just mentioning it is related to the `apply state` or `edit state`?